owncloudApp/src/main/res/values/strings.xml

```diff
@@ -836,4 +839,11 @@
     <string name="content_description_delete_share">Delete share</string>
     <string name="content_description_file_operations">%1$s operations</string>
 
+    <string name="create_shortcut_dialog_title">Create new shortcut</string>
+    <string name="create_shortcut_dialog_url">URL</string>
+    <string name="create_shortcut_dialog_url_information">The URL must start with https:// or http:// so that it can be opened in the browser.</string>
```

Comment:
```suggestion
    <string name="create_shortcut_dialog_url_information">The URL must start with https:// or http:// so that it can be opened in the browser</string>
```

Reply:
Done